docs/content/configurations/docker-images.md

```diff
@@ -17,8 +17,9 @@ golang = "sdsws/golang:1.1"
 ruby = "sdsws/ruby:1.0"
 mysql = "mysql:5.7"
 mongodb = "mongo:4.2.1"
+postgresql ="postgres:latest"
```

Comment:
Change this to `postgresql = "postgres:12.2-alpine"`

Reply:
Fixed . made changes to overview also